resources/configuration.go

```diff
@@ -0,0 +1,32 @@
+package resources
+
+import (
+	"github.com/generationtux/brizo/database"
+	"github.com/jinzhu/gorm"
+)
+
+// Configuration as defined by Brizo.
+type Configuration struct {
+	database.Model
+	Name          string      `gorm:"not null" json:"name"`
+	Value         string      `gorm:"not null" json:"slug"`
```

Comment:
I would name this something like `EnvConfig` or `EnvironmentConfig`, just because configuration is a little generic